hack/nix/spiff/deps.nix

```diff
@@ -0,0 +1,246 @@
+# file generated from Gopkg.lock using dep2nix (https://github.com/nixcloud/dep2nix)
```

Comment:
I guess this file needs to be updated basically every time when go.mod is updated, am I right?

Reply:
If spiff is updated to the newer go module support we can remove this file and use buildGoModule instead which uses a single hash value over all dependencies.
In nix, everything is about reproducibility. You can have multiple versions of any library/program installed in parallel, which helps a lot when testing different environments.